89.md

```diff
@@ -0,0 +1,76 @@
+NIP-89
+======
+
+Bitcoin Transaction Broadcasting Events
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains encoded bitcoin transactions.
```

Comment:
Do you want to use the term 'package relay' anywhere in this NIP? I think it would be nice.

Reply:
Referred to them as transaction packages in the recommendation section. Probably can add more color though

Reply:
Ah yes, missed that. Nostr package relay sounds good too though.

Reply:
Add 'package' to the NIP title? In my view, this is a great selling point for the NIP.